### ISSUE TITLE
Discovery Queue Reward Detection/Prompt Doesn't Work Anymore

### ISSUE THREAD

**Issue by PotcFdk** [PotcFdk/SteamDiscoveryQueueAutoSkipper#15]
There is detection code for checking if processing a queue will yield trading cards:

https://github.com/PotcFdk/SteamDiscoveryQueueAutoSkipper/blob/2763bebcb70049e1c70e8de570ef3056f614cca5/SteamDiscoveryQueueAutoSkipper.user.js#L320-L336

It checks if getQueueCount returns > 0. getQueueCount tries to find a `subtext` class tag, which doesn't exist anymore:

https://github.com/PotcFdk/SteamDiscoveryQueueAutoSkipper/blob/2763bebcb70049e1c70e8de570ef3056f614cca5/SteamDiscoveryQueueAutoSkipper.user.js#L111-L143

This seems to be broken as Valve changed the website probably some time throughout this year.

**Comment by deuxdoom** [comment]
Maybe Valve changed discovery queue system 2024 winter sale.

**Comment by sigma67** [comment]
discovery queue is now a popup for me

![image](https://github.com/user-attachments/assets/fad6b46e-1e0b-4710-8ab6-d0832a60ee73)

**Comment by PotcFdk** [comment]
Yes, it seems like there are now two forms of discovery queues. The old one at the old url still worked the last time I tried it, but the new one requires new logic. Since this issue is about the detection/prompt, support for the new queue form will be tracked in #16.
